src/sprites/rendered-target.js

```diff
@@ -390,6 +390,39 @@ class RenderedTarget extends Target {
     }
 
     /**
+     * Delete a costume by index.
+     * @param {number} index Costume index to be deleted
+     */
+    deleteCostume (index) {
+        const originalCostumeCount = this.sprite.costumes.length;
+        if (originalCostumeCount === 1) return;
+
+        this.sprite.costumes = this.sprite.costumes
+            .slice(0, index)
+            .concat(this.sprite.costumes.slice(index + 1));
+
+        if (index === originalCostumeCount - 1) {
+            this.setCostume(index - 1);
+        } else {
+            this.setCostume(index);
+        }
```

Comment:
This still seems to do an odd thing - say you have 5 costumes:
```
Costume 1
Costume 2
Costume 3
Costume 4   <-- current costume
Costume 5

Costume 1
-Costume 2- <-- deleted costume
Costume 3
Costume 4   <-- current costume
Costume 5

Behavior                           Expected
Costume 1                          Costume 1                      
Costume 3   <-- current costume    Costume 3                      
Costume 4                          Costume 4   <-- current costume
Costume 5                          Costume 5                      
```
If we allow deleting costumes other than the currently selected one, this is weird behavior — we should maintain the current costume if we delete an index < than the current costume index.